assets/app/mail/reset.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Reset < Snabberb::Component
+  needs :user
+  needs :hash
+  needs :base_url
+
+  def render
+    h(:div, [
+      h(:div, "Hello #{@user['name']},"),
+      h(:br),
+      h(:div, "You've requested a password reset at #{Time.now}"),
+      h(:br),
+      h(:div, "Here is your temporary password: #{@hash}"),
+      h(:text, 'Please '),
+      h(:a, { attrs: { href: "#{@base_url}/reset?user_id=#{@user['id']}" } }, 'click here'),
```

Comment:
there's no reason to have the user_id in a param here since they will log in with their email anyways